include/fastrtps/rtps/messages/RTPSMessageGroup.h

```diff
@@ -98,10 +101,10 @@ class RTPSMessageGroup
         bool add_gap(std::set<SequenceNumber_t>& changesSeqNum, const std::vector<GUID_t>& remote_readers,
                 const LocatorList_t& locators);
 
-        bool add_acknack(const GUID_t& remote_writer, SequenceNumberSet_t& SNSet,
+        bool add_acknack(const std::vector<GUID_t>& remote_writers, SequenceNumberSet_t& SNSet,
```

Comment:
Now this functions accept a vector, but it only sends the acknack to the first endpoint. It will be usefull to document this for developer purpose, to avoid any error.